web/src/routes/LoginPage.tsx

```diff
@@ -88,23 +91,53 @@ const Button = styled.button`
   }
 `;
 
-export default function MainPage() {
+export default function LoginPage() {
   const [theme, toggleTheme] = useTheme();
   const navigate = useNavigate();
+  const [inputEmail, setInputEmail] = useState('');
+  const [inputPassword, setInputPassword] = useState('');
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+  };
+
+  // It will have to Add connect User DataBase - Email & Password
+  const CheckLogin = (email: string, password: string) => {
+    if (isEmailEmpty(email) || isPasswordEmpty(password)) {
```

Comment:
Why do you need `isEmailEmpty()` and `isPasswordEmpty()` functions? They're doing the same thing if the string is empty string or not. So please just make one function that check if the it's empty string or not.

See https://stackoverflow.com/a/154068/17320013